### PR TITLE
Update contribution guide links

### PR DESCRIPTION
**Pull request by dimitraz**
Update contributing guide links

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -43,8 +43,7 @@ not already exist.
 ## Submitting changes
 
 * Push your changes to a topic branch in your fork of the repository.
-* Submit a pull request to the repository in the [Aerogear GitHub organization]
-  (https://github.com/aerogear) and choose branch you want to patch 
+* Submit a pull request to the repository in the [Aerogear GitHub organization](https://github.com/aerogear) and choose branch you want to patch
```
Comment (laurafitzgerald) [inline .github/CONTRIBUTING.md]: we could add the aerogearcatlog org here as well, as both are mentioned on https://aerogear.org/community/#contact

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -73,4 +72,4 @@ language we use. View them [here](https://github.com/fheng/best_practice).
 
 * [General GitHub documentation](http://help.github.com/)
 * [GitHub pull request documentation](https://help.github.com/articles/about-pull-requests/)
-* [Read the Issue Guidelines by @necolas](https://github.com/necolas/issue-guidelines/blob/master/CONTRIBUTING.md) for more details
\ No newline at end of file
+* [Read the Issue Guidelines by @necolas](https://github.com/necolas/issue-guidelines/blob/master/CONTRIBUTING.md) for more details
```
Comment (laurafitzgerald) [inline .github/CONTRIBUTING.md]: This is probably fine now but I think we'll update it to link to some content around how we manage github issues coming out of the discussion at the community meeting earlier today.

**Comment by laurafitzgerald** [comment]
also follow on question: will this be updated as part of the work going on to sync up contributing guides and issue templates as mentioned on [this mail thread](https://groups.google.com/forum/#!topic/aerogear/jRYyijix1uw)?

File: .github/CONTRIBUTING.md (hunk 2 above)
Comment (dimitraz) [reply .github/CONTRIBUTING.md]: Ok, will wait for that meeting to update it and merge this pr for now

**Comment by dimitraz** [comment]
@laurafitzgerald I think it should be. Going to merge this PR for now and we can create an item in the aerogear community trello board for that work

**Comment by laurafitzgerald** [comment]
@dimitraz could you link the card here once it's created. thanks.

**Pull request merged**